wallet-new/integration/Util.hs

```diff
@@ -166,3 +169,12 @@ noLongerThan action maxWaitingTime = do
     case res of
         Left _  -> return Nothing
         Right a -> return $ Just a
+
+
+-- | Output for @Text@.
+printT :: Text -> IO ()
+printT = putStrLn
```

Comment:
@MarcFontaine just as a side note, there is `putText` from Universum which does exactly this 😉